api/rest/v1/bulk_find_entities.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: default
+title: Find DCIDs
+parent: REST (v1)
+grand_parent: API
+nav_order: 1
+published: true
+permalink: /api/rest/v1/bulk/find/entities
+---
+
+# /v1/bulk/find/entities
+
+Find the [DCID](/glossary.html#dcid) of an entity.
+
+Given the name of the entity, this endpoint searches for an entry in the Data Commons knowledge graph and returns the DCID of the best match. For example, you could query for "San Francisco, CA" or "San Francisco" to find that its DCID is `geoId/0667000`. You can also provide the type of entity (country, city, state, etc.) to disambiguate (Georgia the country vs. Georgia the US state).
```

Comment:
Change "name" to "description"? The request message has a field of "description" instead of "name".

Reply:
Again, for a given description, we might find multiple entities in our KG, thus return multiple possible DCIDs.

Also note that, if multiple DCIDs are turned, the first one is the most likely one given the available info.

Reply:
Added those details.